src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
please extract new Random to class variable

Reply:
not fixed

Reply:
Example
![image](https://user-images.githubusercontent.com/12864128/223416026-1ad872ea-4c1a-4ac9-a456-f80b93d6fcb6.png)